bokeh/embed.py

```diff
@@ -34,15 +42,19 @@ def components(plot_object, resources):
     Args:
         plot_object (PlotObject) : Bokeh object to render
             typically a Plot or PlotContext
-        resources (Resources, optional) : BokehJS resources config
-
+        resources : Deprecated argument
     Returns:
         (script, div) : UTF-8 encoded
 
     '''
+    
+    if resources is not None:
+        warn('the ``resources`` argument is deprecated and will be removed in'
+             'a future version.', DeprecationWarning, stacklevel=2)
+    
```

Comment:
Regarding the non-concatenated JS my understanding is that is simply not possible with browserify build. However, you can use source maps to be able to debug at the coffeescript file level inside the browser. I tried this yesterday and it worked OK in chrome, but not in safari, at least  out of the box. Also the "debug" unminfied `bokeh.js` is pretty enormous. I think our current build (since we are not atm vendoring pre-minified dependencies) might be adding source maps for all the dependencies as well. @tswicegood can you comment? Is there any way to refine or improve that behavior?